kafka/src/test/java/com.expedia.adaptivealerting.kafka/KafkaDetectorManagerTest.java

```diff
@@ -48,6 +48,7 @@
     private static final String KAFKA_KEY = "some-kafka-key";
     private static final String INBOUND_TOPIC = "mapped-metrics";
     private static final String OUTBOUND_TOPIC = "anomalies";
+    private static final String INVALID_VALUE = "invalid-value";
```

Comment:
Should this be `INVALID_METRIC_VALUE` not 100% sure about its usage.

Reply:
Will clarify it, thanks.

Reply:
Updated.